soccer/Robot.hpp

```diff
@@ -530,6 +532,9 @@ class OurRobot : public Robot {
     friend class MotionControl;
 
 private:
+    mutable RJ::Time _lastBallSense;
+    const long _lostBallDuration = 200000000;
```

Comment:
Also, 0.2 seconds seems a little high for a first pass. Could we make this <=0.1 seconds and increase it if it helps?

I would also love for this to be tested on real robots if you get the chance, the ball dynamics are very different, especially up close.